src/app/page.tsx

```diff
@@ -1,222 +1,350 @@
+    );
+    setTotalAmount(
+      mode === "saving" ? calculatedTotal : amount + calculatedInterest
+    );
   }, [amount, interestRate, years, compounding, mode]);
-  
+
   return (
-    <div className="bg-white p-6 rounded-lg shadow-lg max-w-3xl mx-auto">
-      <h2 className="text-2xl font-bold text-center text-black mb-2">Understanding Money Building Blocks</h2>
-      <p className="text-center text-gray-600 mb-6">At its core, money has a price - we call this the interest rate.</p>
-      
-      <div className="flex flex-col md:flex-row gap-8 mb-8">
-        {/* Left side explanation */}
-        <div className="flex-1 bg-stone-50 p-4 rounded-lg">
-          <h2 className="text-lg font-semibold text-stone-600 mb-3">The Concept of Interest</h2>
-          <p className="text-gray-700 mb-3">
-            When you <span className="font-medium">save money</span>, you are essentially a lender, 
-            and you receive interest from those using it.
-          </p>
-          <p className="text-gray-700 mb-3">
-            When you <span className="font-medium">borrow</span>, you are paying for the privilege 
-            of using someone else`&#39;`s money.
-          </p>
-          <p className="text-gray-700 italic">
-            In this interactive calculator, you can see how interest works in both scenarios.
-          </p>
-        </div>
-        
-        {/* Right side image */}
-        <div className="flex-1 flex items-center justify-center">
-          <div className="w-full h-full bg-stone-100 rounded-lg flex items-center justify-center">
-            <svg width="200" height="200" viewBox="0 0 200 200">
-              {/* Money stacks */}
-              <rect x="30" y="120" width="60" height="60" fill="#008566" rx="4" />
-              <rect x="35" y="115" width="50" height="10" fill="#006F54" rx="2" />
-              <text x="60" y="155" textAnchor="middle" fontSize="14" fill="white" fontWeight="bold">$</text>
-              
-              {/* Interest visualization */}
-              <rect x="110" y="120" width="60" height="60" fill={mode === 'saving' ? "#B6B1A9" : "#B1040E"} rx="4" />
-              <rect x="115" y="115" width="50" height="10" fill={mode === 'saving' ? "#7F7776" : "#ef4444"} rx="2" />
-              <text x="140" y="155" textAnchor="middle" fontSize="14" fill="white" fontWeight="bold">
-                {mode === 'saving' ? "+" : "-"}%
-              </text>
-              
-              {/* Labels */}
-              <text x="60" y="195" textAnchor="middle" fontSize="12" fill="#1f2937">Principal</text>
-              <text x="140" y="195" textAnchor="middle" fontSize="12" fill="#1f2937">
-                {mode === 'saving' ? "Interest Earned" : "Interest Paid"}
-              </text>
-            </svg>
+    <div className="bg-white p-6 max-w-5xl mx-auto">
+      <h2 className="poppins-bold text-[20px] lg:text-[50px] font-bold text-black mb-2">
```

Comment:
Update this to something like `<h2 className={ `${poppins.className} text-[20px] lg:text-[50px] font-bold text-black mb-2` }>`